scripts/lib/auth-helper.ts

```diff
@@ -0,0 +1,66 @@
+import { authenticate } from '@google-cloud/local-auth';
+import fs from 'fs';
+import { AuthClient, OAuth2Client } from 'google-auth-library';
+import { google } from 'googleapis';
+
+const TOKEN_PATH = 'secrets/token.json';
+const CREDENTIALS_PATH = 'secrets/credentials.json';
```

Comment:
Might be worth doing these as

```ts
const TOKEN_PATH = path.join(__dirname, '../../secrets/token.json');
```

Just to be robust against the script being run from an unexpected working directory.